src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -145,39 +146,47 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			// For Escape - the focused element is the div.notebook-preview or textarea.inputarea of the cell, so we need to make sure that it is a descendant of the current active cell
 			//  on the current active editor.
 			const activeCellElement = this.container.nativeElement.querySelector(`.editor-group-container.active .notebook-cell.active`);
+			let handled = false;
```

Comment:
might be good to leave a comment on what this represents or rename it to something like `handledKeyEvent`